packages/lib/component-utilities/src/echarts.js

```diff
@@ -11,7 +11,7 @@ import * as chartWindowDebug from './chartWindowDebug';
  * } ActionParams
  */
 
-const ANIMATION_DURATION = 500;
+const ANIMATION_DURATION = navigator.webdriver ? 0 : 500;
```

Comment:
Is this disabling animations in tests? If so, I don't think we should make this change. Testing in general is most effective when the code we're testing matches the code users are running. Adding differences like this reduces the effectiveness of testing.

If you added this to avoid adding delays to the tests, instead you should look for another way to know that the chart is fully mounted/animated (like a callback/flag) if that is important for the tests

Reply:
I did that so I could use the `finished` event to figure out chart rerenders - there's  a `rendered` event I'm trying but it seems like it might not be very consistent